test/integration/features/proxy.feature

```diff
@@ -0,0 +1,31 @@
+@proxy @linux
+Feature: Test the proxy
+
+  The user try to use crc behind proxy. he/she expect the
+  crc start happen successfully and able to deploy the app and check its
+  accessibility.
+
+  Scenario: Setup the proxy container using podman
+    Given executing "sudo podman run --name squid -d -p 3128:3128 quay.io/crcont/squid" succeeds
+
+  Scenario: Start CRC
+    Given executing "crc setup" succeeds
+    And  executing "crc config set http-proxy http://192.168.130.1:3128" succeeds
+    Then executing "crc config set https-proxy http://192.168.130.1:3128" succeeds
+    When starting CRC with default bundle succeeds
+    Then stdout should contain "Started the OpenShift cluster"
+    And executing "eval $(crc oc-env)" succeeds
+    When with up to "4" retries with wait period of "2m" command "crc status --log-level debug" output matches ".*Running \(v\d+\.\d+\.\d+.*\).*"
+    Then login to the oc cluster succeeds
+
+  Scenario: Remove the proxy container and host proxy env (which set because of oc-env)
+    Given executing "sudo podman stop squid" succeeds
+    And executing "sudo podman rm squid" succeeds
```

Comment:
Is this intentional that you did not use `podman run --rm`, which would avoid the need for this `podman rm`?

Reply:
@cfergeau `--rm` option doesn't behave well with `-d` the way we are running. Also old version of podman doesn't support to use `--rm` with `-d` option (what docker used to do).

```
$ podman run --name squid --rm -d -p 3128:3128 quay.io/crcont/squid 
42560e1d8a7a157ced7fa393c1c475715dc532af70c292c1315009d6c77503e5

$ podman stop squid
Error: container 42560e1d8a7a157ced7fa393c1c475715dc532af70c292c1315009d6c77503e5 does not exist in database: no such container
```